### PR TITLE
fix(breakage): gmarket.co.kr

### PR DESCRIPTION
**Pull request by seia-soto**
Fixes wide breakage in `gmarket.co.kr`. Blocking `script.gmarket.co.kr/js/header/statsna.js` will result to undefined reference in their inline scripts: `clickPdsLog` and etc when we want to do some actions: adding reviews or inquiries. Instead, we tend to block the report hosts instead.

### CODE REVIEW

File: youslist.txt [hunk 1]
```diff
@@ -7503,6 +7505,8 @@ namu.wiki##:xpath(//div/h1[not(./small)]/following-sibling::div[1]/following-sib
 @@||cdn.kbmaeil.com/bannerpop/uploads/js/*.js^$domain=kbmaeil.com
 @@||googleads.g.doubleclick.net/pagead/ads?*&crui=*
 @@||mindpost.or.kr/script/bannerpop.js^$domain=mindpost.or.kr
+@@||script.gmarket.co.kr/js/common/pds_gmkt.js
+@@||script.gmarket.co.kr/js/header/statsna.js
```
Comment (yous) [inline youslist.txt:7509]: Will this be enough, or other sites also use these scripts?

```suggestion
@@||script.gmarket.co.kr/js/common/pds_gmkt.js^$domain=gmarket.co.kr
@@||script.gmarket.co.kr/js/header/statsna.js^$domain=gmarket.co.kr
```

Comment (seia-soto) [reply youslist.txt:7509]: If that covers all subdomain then that looks good to me.

File: youslist.txt [hunk 2]
```diff
@@ -935,6 +935,7 @@
 ||assoc-amazon.com/s/ads.js^
 ||async01.admantx.com/admantx/service^
 ||at.atwola.com/addyn/3.0/
+||ats.auction.co.kr^domain=auction.co.kr
```
Comment (yous) [inline youslist.txt:938]:
```suggestion
||ats.auction.co.kr^$domain=auction.co.kr
```

File: youslist.txt [hunk 3]
```diff
@@ -1788,6 +1789,8 @@
 ||mobile.newsis.com/direct_copy2.html^$domain=mobile.newsis.com
 ||moeni.org/img/sponsor/$domain=moeni.org
 ||moja-ostroleka.pl/uploads/banery/$domain=moja-ostroleka.pl
+||montelena-rcv.gmarket.co.kr^
+||montelena.auction.co.kr^domain=auction.co.kr
```
Comment (yous) [inline youslist.txt:1793]:
```suggestion
||montelena-rcv.gmarket.co.kr^$domain=gmarket.co.kr
||montelena.auction.co.kr^$domain=auction.co.kr
```

File: youslist.txt [hunk 4]
```diff
@@ -1909,13 +1912,16 @@
 ||pcgameshardware.de/webservices/hitAlert.cfc^$domain=pcgameshardware.de
 ||pcwelt.de/js/adserver_footer.js^$domain=pcwelt.de
 ||pd4-imp.revsci.net/rtbads/pq^
+||pds.auction.co.kr^domain=auction.co.kr
```
Comment (yous) [inline youslist.txt:1915]:
```suggestion
||pds.auction.co.kr^$domain=auction.co.kr
```

File: youslist.txt [hunk 5]
```diff
@@ -1909,13 +1912,16 @@
...
 ||pds.dailypharm.com/ADManager_2017/$domain=dailypharm.com|m.dailypharm.com
 ||pds.medicaltimes.com/Banner/$domain=medicaltimes.com
 ||ph.kado.net/adcontent/script/*.js^$domain=kado.net
 ||ph.sisain.co.kr/adcontent/script/*.js^$domain=sisain.co.kr
 ||ph.spotvnews.co.kr/adcontent/script/*.js^$domain=m.spotvnews.co.kr|spotvnews.co.kr
 ||photo.coolenjoy.net/banner/$domain=coolenjoy.net|cooln.kr|on.coolenjoy.net
 ||physicsworld.com/wp-content/themes/iopp/assets/js/realmedia.js^$domain=physicsworld.com
+||pics.auction.co.kr/common/blank.gif$domain=auction.co.kr
+||pics1.iacstatic.co.kr/common/blank.gif$domain=auction.co.kr
```
Comment (yous) [inline youslist.txt:1924]:
```suggestion
||pics.auction.co.kr/common/blank.gif^$domain=auction.co.kr
||pics1.iacstatic.co.kr/common/blank.gif^$domain=auction.co.kr
```

File: youslist.txt [hunk 6]
```diff
@@ -2243,6 +2249,8 @@
 ||uplus.kr/common/m_js/mobile_html_web*.js^
 ||urldelivery.com/watch.
 ||util.simply-hentai.com/prod/*?d=$domain=simply-hentai.com
+||uts.gmarket.co.kr^
+||utsgw.auction.co.kr^domain=auction.co.kr
```
Comment (yous) [inline youslist.txt:2253]:
```suggestion
||uts.gmarket.co.kr^$domain=gmarket.co.kr
||utsgw.auction.co.kr^$domain=auction.co.kr
```

**Review by yous — COMMENTED** [review]
Thanks!

**Comment by seia-soto** [comment]
Oh, thanks. I forgot to add a lot of syntax sugars.

**Comment by yous** [comment]
Thank you!

**Pull request merged**